src/main/java/edu/ucsb/cs156/happiercows/controllers/CommonsController.java

```diff
@@ -77,14 +77,20 @@ public ResponseEntity<String> updateCommons(
       updated = new Commons();
       status = HttpStatus.CREATED;
     }
+     if(params.getDegradationRate() < 0) { //disallowing negative values for degradation rate
```

Comment:
It turns out that there is no test, that I can find anyway, that checks that this behavior is in the code.

Specifically lines 80-82 on CommonsController.java
    //  if(params.getDegradationRate() < 0) { //disallowing negative values for degradation rate
    //   updated.setDegradationRate(-1*params.getDegradationRate());
    //  }
    
Please write a test that will fail if these lines are removed.
It should probably be an assertion in the test for update.

(Note that there IS a test for the corresponding lines in create.)

I'm singling these out because they are unusual: it's unusual to silently override user input.  So if we are going to do it, we need to double down and make sure that this is the intended behavior.

I would have preferred throwing an IllegalArgumentException, but if this is the way we are going, let's at least be sure it's tested.